backend/qemu.pm

```diff
@@ -283,6 +283,8 @@ sub _migrate_to_file {
             arguments => {
                 # This is ignored if the compress capability is not set
                 'compress-level' => $compress_level,
+                # Default of 8 causes high load spikes for concurrent runs
+                'compress-threads' => 2,
```

Comment:
@richiejp wdyt? imo compress-threads would then/should be configurable too...with the same or similar treatment as compress_level below...

Reply:
yeah, sure. I guess I assumed QEMU would choose a sensible default for the arch or just overlooked.

Reply:
the default of 8 is documented, and I think it isn't a sensible default for openqa in any case, no matter which architecture we're talking about because it suddenly eightfolds the cpu load on a load that is anyway cpu bound.

Reply:
We could limit it to QEMUCPU so the compression can only produce as much load as the VM/SUT itself could. Users should not give their machines more cores than the worker can handle.

During my testing it didn't make a lot of difference what compression level is being used so setting it to 6 is probably fine.

> on a load that is anyway cpu bound

This isn't true in all cases. For example, most deployments are actually a single worker on a test developer's machine. On all my development machines the workload is I/O bound.

Also a lot of the server grade ARM machines I have seen have 48-96 cores, but poor single core performance, so it makes more sense to me to reduce the number of workers on these machines, but give each worker more cores.